### DISCUSSION TITLE
disable/enable create upload folder

### DISCUSSION THREAD

**Discussion by gogi100** [rejetto/hfs#194] [General]
is it possible to set in the admin panel to choose whether users will be able to create folders and upload folders or not?

**Comment by rejetto** [comment]
i will add a class to buttons so we can hide the buttons you want using css/style

**Comment by rejetto** [comment]
try version [0.43.0-beta5](https://github.com/rejetto/hfs/files/11162764/hfs-windows.zip)
with this code

```
<style>
.upload-folder, .create-folder { display: none }
</style>
```

**Reply by gogi100** [reply]
thanks
its working
